cmd/exchange_wrapper_issues/main.go

```diff
@@ -414,8 +415,9 @@ func testWrappers(e exchange.IBotExchange, base *exchange.Base, config *Config)
 			Response: jsonifyInterface([]interface{}{r7}),
 		})
 
-		var r8 []exchange.TradeHistory
-		r8, err = e.GetExchangeHistory(p, assetTypes[i])
+		r8r := &trade.HistoryRequest{}
```

Comment:
This needs to be populated properly, the report returns contains errors for the wrapper functions now.
For reference, I used the following in all my screenshots posted here:
```
r8r := &trade.HistoryRequest{
			Pair:           p,
			Asset:          assetTypes[i],
			TimestampStart: time.Now().Add(-time.Hour),
		}
```